app/views/forms/_screen.html.erb

```diff
@@ -37,8 +37,101 @@
 
     </div>
 
-    <%# questions on the form %>
-    <%= index_table(Questioning, @form.questionings, :within_form => true) if form_mode != :new %>
+    <div class="questions">
+      <input class="btn btn-primary" type="submit" value="Save"></input>
+
+      <h2>Questions</h2>
+
+      <div class="questions-header" id="questions-header">
+        <span><a href="choose-questions"><i class="fa fa-plus"></i> Add Question</a></span>
+        <span class="q-addGroup"><a><i class="fa fa-folder-open-o"></i> Add Group</a></span>
+      </div>
+
+      <div class="draggable-list-wrapper">
+        <ol class="questions-list ui-sortable">
+          <% @form.questionings.each do |qing| %>
+            <li class="question">
+
+              <%# Standard question? %>
+              <% if qing.standardized? == true %>
+                <div class="q-standardized"><i class="fa fa-certificate"></i></div>
```

Comment:
Do you need the `<div>` here? Can you perhaps just apply your styles to `fa-certificate`? We try to keep things as terse as possible.